src/tests/utils/Processing.test.js

```diff
@@ -0,0 +1,88 @@
+import temp from 'temp';
+import { loadExternalDependenciesPlayground, readConfigPlayground, fillTank } from '../../utils/Processing';
+import { RawFSH } from 'fsh-sushi/dist/import/RawFSH';
+import { FHIRDefinitions } from 'fsh-sushi/dist/fhirdefs';
+import * as loadModule from '../../utils/Load';
+import 'fake-indexeddb/auto';
+
+describe('Processing', () => {
+  temp.track();
+
+  describe('#readConfigPlayground', () => {
+    it('should hard code the preferred config', () => {
+      const config = readConfigPlayground();
+      expect(config.fhirVersion).toEqual(['4.0.1']);
+      expect(config.id).toEqual('fhir.us.minimal');
+      expect(config.FSHOnly).toBe(false);
+    });
+  });
+
+  describe('#loadExternalDependenciesPlayground()', () => {
+    beforeAll((done) => {
+      done();
+    });
+    afterAll((done) => {
+      done();
+    });
+
+    it('should log an error when it fails to make the database', () => {
+      const defs = new FHIRDefinitions();
+      const version = -1;
+      const dependencyDefs = loadExternalDependenciesPlayground(defs, version);
+      return expect(dependencyDefs).rejects.toThrow(TypeError);
+    });
+
+    it('should create a new database when one does not yet exist', async () => {
+      const defs = new FHIRDefinitions();
+      const version = 2;
+      const unzipSpy = jest.spyOn(loadModule, 'unzipDependencies').mockImplementation(() => {
+        return undefined;
+      });
+      const loadDefsSpy = jest.spyOn(loadModule, 'loadDependenciesInStorage').mockImplementation(() => {
+        return undefined;
+      });
+      const loadIntoPlaygroundSpy = jest.spyOn(loadModule, 'loadIntoDefsPlayground').mockImplementation(() => {
+        return undefined;
+      });
+      const dependencyDefs = loadExternalDependenciesPlayground(defs, version);
+      await dependencyDefs;
+      expect(unzipSpy).toBeCalled();
+      expect(loadDefsSpy).toBeCalled();
+      expect(loadIntoPlaygroundSpy).toBeCalled();
+    }, 99999999);
```

Comment:
Is the timeout here and on line 74 still needed? I remember we were messing around with different async and await things, but we might have figured all that out and the timeout `99999999` might not be needed anymore!